.github/workflows/linter.yml

```diff
@@ -0,0 +1,54 @@
+---
+###########################
+###########################
+## Linter GitHub Actions ##
+###########################
+###########################
+
+name: Lint Code Base
+
+#
+# Documentation:
+# https://help.github.com/en/articles/workflow-syntax-for-github-actions
+#
+
+#############################
+# Start the job on all push #
+#############################
+on: [ push, pull_request, workflow_dispatch ]
+
+###############
+# Set the Job #
+###############
+jobs:
+  build:
+    # Name the Job
+    name: Lint Code Base
+    # Set the agent to run on
+    runs-on: ubuntu-latest
+
+    ##################
+    # Load all steps #
+    ##################
+    steps:
+      ##########################
+      # Checkout the code base #
+      ##########################
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      ################################
+      # Run Linter against code base #
+      ################################
+      - name: Lint Code Base
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_PYTHON: true
+          VALIDATE_PYTHON_FLAKE8: true
+          # VALIDATE_PYTHON_PYLINT: false # disable pylint, as we have not configure it
+          # VALIDATE_PYTHON_BLACK: false # same as above
+          PYTHON_FLAKE8_CONFIG_FILE: tox.ini
+          FILTER_REGEX_INCLUDE: azure_functions_worker/.*
+          FILTER_REGEX_EXCLUDE: tests/.*
+          DEFAULT_BRANCH: dev
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
May need to set the GITHUB_TOKEN secrets in This Repo -> Settings -> Secrets.

Reply:
We dont need to pass as the workflow is running within GH.

From the documentation on [How to use the super linter](https://github.com/marketplace/actions/super-linter#how-to-use)?
> NOTE: If you pass the Environment variable GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} in your workflow, then the GitHub Super-Linter will mark the status of each individual linter run in the Checks section of a pull request. Without this you will only see the overall status of the full run. There is no need to set the GitHub Secret as it is automatically set by GitHub, it only needs to be passed to the action.